addons/binding/org.openhab.binding.fritzaha/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="fritzaha"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Supported Fritz! devices and features -->
+    <thing-type id="FRITZ_DECT_200">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="fritzaha-bridge" />
+		</supported-bridge-type-refs>
+
+        <label>Fritz!DECT 200</label>
+        <description>Fritz!DECT200 switchable outlet</description>
+
+		<channels>
+			<channel id="temperature" typeId="temperature" />
+			<channel id="energy" typeId="energy" />
+			<channel id="power" typeId="power" />
+			<channel id="switch" typeId="switch" />
+		</channels>
+
+		<config-description>
+			<parameter name="ain" type="text" required="true">
+				<label>AIN</label>
+				<description>The AHA id (AIN) that identifies one specific device.</description>
+			</parameter>
+		</config-description>
+    </thing-type>
+
+    <thing-type id="FRITZ_Powerline_546E">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="fritzaha-bridge" />
+		</supported-bridge-type-refs>
+
+        <label>Fritz!Powerline 546E</label>
+        <description>Fritz!Powerline 546E with switchable outlet</description>
+
+		<channels>
+			<channel id="energy" typeId="energy" />
+			<channel id="power" typeId="power" />
+			<channel id="switch" typeId="switch" />
+		</channels>
+
+		<config-description>
+			<parameter name="ain" type="text" required="true">
+				<label>AIN</label>
+				<description>The AHA id (AIN) that identifies one specific device.</description>
+			</parameter>
+		</config-description>
+    </thing-type>
+
+	<!-- Channel definitions of fritz devices -->
+	<channel-type id="temperature">
+		<item-type>Number</item-type>
+		<label>Actual Temperature</label>
+		<description>Actual measured room temperature</description>
+		<category>Temperature</category>
+		<tags>
+			<tag>temperature</tag>
+		</tags>
+		<state pattern="%.1f °C" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="energy">
+		<item-type>Number</item-type>
+		<label>Energy Consumption</label>
+		<description>Accumulated energy consumption</description>
+		<category>Energy</category>
+		<tags>
+			<tag>energy</tag>
+		</tags>
+		<state pattern="%.2f kWh" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="power">
+		<item-type>Number</item-type>
+		<label>Power Consumption</label>
+		<description>Current power consumption</description>
+		<category>Energy</category>
+		<tags>
+			<tag>energy</tag>
+		</tags>
+		<state pattern="%.2f W" readOnly="true">
+		</state>
+	</channel-type>
+    <channel-type id="switch">
+        <item-type>Switch</item-type>
+        <label>Switched outlet</label>
```

Comment:
I'd simply put "Power" here - rendered with a toggle, the meaning should be clear to the user